docker/amppkg.example.toml

```diff
@@ -0,0 +1,129 @@
+# extension to the OCSP responses from the URI mentioned in its Authority
+# Information Access extension. (See https://goo.gl/JQiyNs item 7.4.)
+#
+# To mitigate the risk of an attacker gaining access to your private key
+# through a vulnerability in this packager, you should generate a new
+# certificate/key pair for the domain(s) you wish to sign packages with. This
+# allows you to revoke that certificate without affecting your normal
+# web-serving traffic.
+CertFile = '/go/src/github.com/ampproject/amppackager/testdata/b1/fullchain.cert'
+
+# The path to the PEM file containing the private key that corresponds to the
+# leaf certificate in CertFile.
+KeyFile = '/go/src/github.com/ampproject/amppackager/testdata/b1/server.privkey'
+
+# The path to a file where the OCSP response will be cached. The parent
+# directory should exist, but the file need not. If this is a network-mounted
+# file, it should support shared/exclusive locking.
+OCSPCache = '/tmp/amppkg/ocsp'
+
+# This is a simple level of validation, to guard against accidental
+# misconfiguration of the reverse proxy that sits in front of the packager.
+#
+# You must specify at least one URLSet; you may specify multiple. Each one must
+# specify a Sign pattern and may specify a Fetch pattern. When the packager
+# receives a request for a package, it will first validate that the requested
+# fetch/sign URL pair matches at least one of the given URLSets.
+[[URLSet]]
+  # What URLs are allowed to show up in the browser's URL bar, when served from
+  # the AMP Cache. By default, the URL that the frontend requests to sign is
+  # also the URL where the packager fetches it. For extra flexibility, see
+  # URLSet.Fetch below.
+  [URLSet.Sign]
+    # The scheme of the URL must be https. There is no way to configure this.
+    # The `user:pass@` portion is disallowed. There is no way to configure this.
+
+    # The domain to limit signed URLs to. An exact string match. The
+    # certificate must cover this domain.
+    Domain = "example.com"
+
+    # A full-match regexp on the path (not including the ?query).
+    # Defaults to ".*". The below example allows paths starting with /world/.
+    # PathRE = "/world/.*"
+
+    # A list of full-match regexps, carving out exclusions to the above PathRE.
+    # Examples of paths you might want to exclude:
+    #   * Personalized content, such as user settings pages. Signed exchanges
+    #     are cached globally and served to all users. (Personalization that
+    #     happens at runtime via JS is fine.) This provides defense-in-depth,
+    #     in addition to the Cache-Control: public check.
+    #   * User-generated content, such as forums. For instance, if there's a
+    #     chance of an XSS vulnerability in your templating library, the impact
+    #     of such event is higher here: even after you've fixed the bug, caches
+    #     may serve your signed packages for up to 7 days.
+    PathExcludeRE = []
```

Comment:
You can remove this or comment it out; it's the default. (I included the fix to the other example.toml in my PR to you.)

Reply:
removed.